creator-node/tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "target": "es5",
+    "module": "commonjs",
+    "lib": [
+      "esnext"
+    ],
+    "outDir": "./build",
+    "allowJs": true,
+    "downlevelIteration": true,
+    "isolatedModules": true,
```

Comment:
I think you can drop 
```
"isolatedModules": true,
```

because it's only necessary when using something other than `tsc` to compile. https://www.typescriptlang.org/tsconfig#isolatedModules

Which means we don't need the `export {}` in index.ts

Reply:
nodemon uses ts-node to do hot reload. would we be able to remove it in that case @sliptype?

Reply:
I believe so, yes. I ran this without it and it worked fine

Worst case would be a runtime error on local on some minor TS edgecases